### PR TITLE
Move intro to spec, add sequence diagram, and other minor review feedback items

### PR DESCRIPTION
**Pull request by mbarkley**
In this PR:
* Most of introduction moved into the spec
* Background section of introduction moved to the FAQ
* "Clients" added to list of main concepts in spec (in introduction)
* Short intro added to Visas Issuer conformance section
* Sequence diagram summarizing full flow added before "Flow of Assertions"

### CODE REVIEW

**Comment by andrewpatto** [comment]
Rendered link https://ga4gh.github.io/data-security/1.2-draft-dg-intro-feedback/

File: AAI/AAIConnectProfile.md [hunk 1]
```diff
@@ -213,7 +272,73 @@ Internet Assigned Numbers Authority
...
 
-## Flow of Assertions
+## Overview of Interactions
+
+### Full Login and Token Exchange Interaction in AAI/Passport 1.2
+
+In the full token exchange flow recommended in this document, the client does not ever distribute the initial
+*Passport-Scoped Access Token* to other services. A token exchange operation is executed by the client, in
+exchange for a *Passport* JWT that may be used downstream to access resources. In this example flow, the
+*Passport* is included as authorization in the POST to a Clearinghouse that holds data.
```
Comment (andrewpatto) [inline AAI/AAIConnectProfile.md:282]: in the POST to a *Passport Clearinghouse* that holds data

**Comment by andrewpatto** [comment]
As a generic comment - we now have an Abstract *and* and Introduction/Tech Summary in the same document - and whilst it was ok when these were two different documents, now I feel that their content are
- both repetitive..
- but also different..
So both the abstract and bits of the intro say the same thing - but say it using very different language and sentence structure.. maybe just some alignment of language and sentence structure between these?
Maybe strip back the abstract so it is super minimal - even more than it is..

**Comment by andrewpatto** [comment]
That said, I approve of doing the moves of this content into the main spec and removing the introduction document

**Review by andrewpatto — APPROVED** [review]

**Review by TomConner — APPROVED** [review]

**Comment by martin-kuba** [comment]
I fixed some broken links, removed "OIDC Relying Party" from Passport Clearinghouse definition, and added missing requirement for conformant signing algorithms to "Conformance for Visa Issuers".

**Review by martin-kuba — APPROVED** [review]

**Pull request merged**